test/WebJobs.Script.Tests/Diagnostics/SanitizerTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Azure.WebJobs.Logging;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Diagnostics
+{
+    public class SanitizerTests
```

Comment:
I forgot about it until right now. But any Sanitizer changes need to go into the SDK as well: https://github.com/Azure/azure-webjobs-sdk/blob/dev/src/Microsoft.Azure.WebJobs.Shared/Sanitizer.cs. 

And there are some tests there: https://github.com/Azure/azure-webjobs-sdk/blob/dev/test/Microsoft.Azure.WebJobs.Host.UnitTests/Loggers/WebJobsSanitizingInitializerTests.cs

You can move these tests there and we don't need any unit tests in this project.

A little background -- we didn't want to make this public b/c it's a tough contract to uphold and we didn't want anyone else using it. So now we have a weird copied file that we need to update in both places.